aten/src/ATen/native/native_functions.yaml

```diff
@@ -12273,6 +12273,21 @@
   dispatch:
     CompositeExplicitAutograd: _test_warn_in_autograd
 
+# Note: this function is only for testing.
+- func: _test_autograd_multiple_dispatch.one(Tensor self) -> Tensor
+  dispatch:
+    CompositeExplicitAutograd: _test_autograd_multiple_dispatch
+    # this is necessary because NestedTensor has ben removed from the CompositeExplicitAutograd keyset
+    # see Note [NestedTensor Not Included in Backend Keys]
+    NestedTensorCPU, NestedTensorCUDA: _test_autograd_multiple_dispatch
```

Comment:
nit: This can be added on the same line as the CompositExplicitAutograd above.

Reply:
...but (just to state the obvious) definitely keep the comment, i.e. "the NestedTensor keys are necessary because..."